lib/translator/common.cc

```diff
@@ -35,6 +37,25 @@ void SetDebugCallback(void (*callback)(const char*)) {
   debug_callback = callback;
 }
 
+// Returns void* casted to uint64_t.
+// A return value of 0 is equivalent to nullptr.
+uint64_t AllocPages(uint16_t count) {
```

Comment:
I'm guessing this is `uint64_t` because that's what NVMe wants, and we don't need to actually touch memory ourselves? I like that change, to the extent that I'd go ahead and use the same convention in the callbacks, then, unless there's something I'm missing.